### ISSUE TITLE
Should Text and Wording add Japanese as a guardrail language?

### ISSUE THREAD

**Issue by julierawe** [w3c/wcag3#553]
In the January 2026 WCAG 3 survey, Jan McSorley commented on behalf of W3C's Global Inclusion community group:

- 2.2.3 - Clear Language - this is written in a way that appears to suggest that content creators should avoid using Kanji characters (Chinese Characters) and use Kana (Japanese Characters) instead. The global-inclusion group is very concerned about this note because no language is similar to the Japanese language - Japanese uses different scripts - both Kanji and Kana. The approach to Japanese is very different. Depending on the literacy levels, you may not use Kanji. UN 6 languages have nothing similar to Japanese. The way this is organized is biased. The Japanese language needs to be elevated in our guidance so that we can avoid this kind of bias. It is not one of the "guardrail languages" for the internationalization work, but this is an example of why it should be. We need a definition of clear language because it's confusing to know what to do without knowing what it means.

**Comment by murata2makoto** [comment]
Some dyslexic Japanese find Kana more difficult to read than Kana.  Others find Kana easier.  Which of them is complex or unclear, then?
